Tests/RxCocoaTests/UITextField+RxTests.swift

```diff
@@ -46,6 +46,16 @@ final class UITextFieldTests : RxTest {
             XCTAssertEqual(textField.attributedText!, attributedText)
         }
     }
+
+    func test_isSecureTextEntryObserver() {
+        // because of leak in iOS 11.2
```

Comment:
You are referring this leak right ? maybe add a link ?
https://forums.developer.apple.com/thread/94323

Reply:
All tests in this file have similar comment about this leak. To be honest, I don't know which exactly leak is mention, i just implement my test in a similar way